src/sinks/aws_cloudwatch_logs/integration_tests.rs

```diff
@@ -467,7 +467,7 @@ async fn create_client_test() -> CloudwatchLogsClient {
     let endpoint = Some(cloudwatch_address());
     let proxy = ProxyConfig::default();
 
-    create_client::<CloudwatchLogsClientBuilder>(&auth, region, endpoint, &proxy, &None, &None)
+    create_client::<CloudwatchLogsClientBuilder>(&auth, region, endpoint, &proxy, &None, &None, true)
```

Comment:
I think this means we will _only_ test with virtual path style addressing. Could we make sure we have at least one test using each style?

Reply:
I got the integration tests working locally and pushed the changes. I had some incorrect values in the test client configs.

On a side note, I did not end up adding an integration test for virtual host style access. This proved more difficult than initially intended as it requires wildcard DNS which docker compose does not support easily. Localstack does support virtual host based lookups and I was able to set up the sdk to do so, but it fails on resolution of the actual bucket. I think wildcard DNS might be out of the scope of this PR.

What are your thoughts here?

Reply:
Here is the branch I was playing with if interested https://github.com/sam6258/vector/tree/sam6258/vhost-buckets-test